sdks/python/apache_beam/yaml/tests/assign_timestamps.yaml

```diff
@@ -49,36 +49,38 @@ pipelines:
               - {user: "alice", timestamp: 3}
               - {user: "bob", timestamp: 7}
 
+  # TODO - Rethink error handling since schema mismatch in Create transform
+  # isn't allowed anymore.
```

Comment:
If I understand this correctly, `typehints.Union` will be used here when they have different schemas, right?

Reply:
In this particular case it will fail with types mismatch since two different types were used that were not nullable.